Automattic/jetpack
# Boost: Add cache clear on settings change

Fixes #36172

## Proposed changes:

* If minify js, css, or image CDN quality settings are changed, clear the page cache.,

### Other information:


## Jetpack product discussion
None

## Does this pull request change what data or activity we track or use?
None

## Testing instructions:
* Make sure a page is cached by checking the `X-Jetpack-Boost-Cache` header.
* Change exclude settings for Concatenate JS/CSS or Image CDN quality settings.
* Make sure cache gets cleared.
